certif/tests/integration/components/issue-report-modal/non-blocking-technical-issue-certification-issue-report-fields-test.js

```diff
@@ -74,7 +74,7 @@ module(
       assert
         .dom(
           screen.getByText(
-            "Signalement à titre informatif, le problème rencontré n'a pas empêché le candidat de continuer à répondre aux questions. En cas d’incident sur une question focus, merci de vous reporter à la catégorie E10.",
+            "Signalement à titre informatif, le problème rencontré n'a pas empêché le candidat de continuer à répondre aux questions. Pour rappel, un incident technique sur une question doit être signalé en direct pendant la session, depuis l'espace surveillant.",
```

Comment:
Je propose ce changement pour se decorreler de la partie traductions (la verite du texte etant porte par Phrase)

```suggestion
            (t('....technical-information'),
```